### DISCUSSION TITLE
请问为什么服务停止了，机器人还能继续回复

### DISCUSSION THREAD

**Discussion by Ancss** [zhayujie/chatgpt-on-wechat#571] [Q&A]
![image](https://user-images.githubusercontent.com/61501274/227459374-540a7d19-d811-4967-8c96-05cb7a8c9bac.png)
![image](https://user-images.githubusercontent.com/61501274/227459449-2981401b-f60e-4c25-a150-45b5f77ff28d.png)
当前服务是停止了吗

**Comment by zhayujie** [comment]
readme有相关说明， 你是nohup 托管在后台运行的， 需要查看进程后 kill 掉
